broker-cli/proto/broker.proto

```diff
@@ -1787,6 +1787,20 @@ message ChangeWalletPasswordRequest {
   string new_password = 3;
 }
 
+/**
+ * Request to recover a wallet
+ */
+message RecoverWalletRequest {
+  // Currency of wallet
+  string symbol = 1;
+  // Password from the existing wallet
+  string password = 2;
+  // 24 word mnemonic seed
+  repeated string seed = 3;
+  // True/False signifying the use of a backup file for committed funds
```

Comment:
is this too tied to LND?

Reply:
That is a good Q. I made a few changes to mitigate this. We'll refer to this flag as `use_backup` not backup file and i've updated documentation to specify when something is related to lnd or not.